Resources/translations/validators.hu.xliff

```diff
@@ -3,9 +3,14 @@
     <file source-language="en" target-language="hu" datatype="plaintext" original="validators.en.xliff">
         <header/>
         <body>
-            <trans-unit id="26" resname="error.uniq_url">
+            <trans-unit id="error.uniq_url" resname="error.uniq_url">
                 <source>error.uniq_url</source>
-                <target>A(z) '%url%' URL már társítva lett egy másik oldalhoz</target>
+                <target>A(z) '%url%' URL már társítva lett egy másik oldalhoz.</target>
+            </trans-unit>
+            <trans-unit id="error.uniq_url.parent_unselect" resname="error.uniq_url.parent_unselect">
+                <source>error.uniq_url.parent_unselect</source>
+                <target>The root URL '/' is already associated with another page of this site, please select a parent.
```

Comment:
Yeah that's strange. It will make symfony shut about missing translations although they really are… but maybe it's to prevent folks that test for missing translations in their functional tests to be annoyed because of us?